app/wp-content/themes/kstg/page-home.php

```diff
@@ -10,6 +10,11 @@
 <?php get_header(); ?>
 <div class="homepage-wrapper">
   <section class="main">
+    <div class="homepage-banner">
+      <p class="uppercase">Kevin Smith Transportation Group</p>
```

Comment:
Do you want to hardcode this? Not sure how often it would be updated, but you could pretty easily add some custom fields to the homepage template if you wanted...